src/core.e/Any-iterable-methods.pm6

```diff
@@ -7,3 +7,10 @@ multi sub rotor(**@cycle-and-thing, *%_) {
     @cycle-and-thing.tail.rotor(@cycle-and-thing.head(*-1), |%_)
 }
 
+multi sub hyper(Iterable:D \thing, *%_) {
```

Comment:
With new-disp, `\` doesn't bring in any performance gain anymore. And, if memory serves me right, a scalar parameter could even be better.

Reply:
Problem is, that this containerizes: `sub a(Iterable:D $a) { dd $a }; a [1,2,3];  # $[1, 2, 3]`.  Which may or may not be an issue in the receiving sub.  Compare this with `sub a(Iterable:D \a) { dd a }; a [1,2,3]; # [1, 2, 3]` and `sub a(Iterable:D $a is raw) { dd $a }; a [1,2,3];  # [1,2,3]`.

Reply:
I know. But in this particular case there is no difference if the `thing` is containerized, neither `race` nor `hyper` on `Iterable` care about this. And as much as I saw in other comments, a scalar is currently preferable over the raw form.

Reply:
Indeed.  I just wanted the op to be aware of the difference between the various ways.  :-)